### DISCUSSION TITLE
Is it intended that I should be able to edit someone else's comment?

### DISCUSSION THREAD

**Discussion by mike-mike-mike-mike-mike** [community/community#16158] [Projects and Issues]
I'm relatively new to github, but when I was reviewing some PRs, I noticed that:

1. I am able to edit someone else's comment with no restrictions
2. There's no audit trail that would show I had done such a thing

This was in a company repository, so it could be someone isolated to that use case, but it struck me as odd (and slightly concerning).

**Comment by Nothing4You** [comment]
https://docs.github.com/en/communities/moderating-comments-and-conversations/managing-disruptive-comments#editing-a-comment

> Anyone with write access to a repository can edit comments on issues, pull requests, and commits.

https://docs.github.com/en/communities/moderating-comments-and-conversations/tracking-changes-in-a-comment

> Anyone with read access to a repository can view a comment's edit history.

https://docs.github.com/en/communities/moderating-comments-and-conversations/managing-disruptive-comments#redacting-sensitive-information

> Comment authors and anyone with write access to a repository can also delete sensitive information from a comment's edit history.

audit trail is on the comment itself, next to the time it was created.

**Reply by Inna-Bogush** [reply]
> audit trail is on the comment itself, next to the time it was created.

only if the comment is unresolved

**Comment by juliettech13** [comment]
Yeah - agreed with @mike-mike-mike-mike-mike ! I've been able to do the same, makes no sense why I'd have such permission. Especially when I can't even comment on my own behalf for such comments. So bizzarre.

**Comment by bh-dremio** [comment]
There is no audit trail on Pull Request Reviews which can be edited by anyone. 

I can edit someone else's review and the [GitHub API to list reviews](https://docs.github.com/en/rest/pulls/reviews?apiVersion=2022-11-28#list-reviews-for-a-pull-request) will reflect the new edited review body with no indication that any edits occurred. There isn't even an "updated_at" field that I can use to compare to the "submitted_at" field.

Currently trying to write a bot that uses review comments and I cannot reliably determine if the review body was written by the review submitter. This is crazy, and very discouraging functionality -_-

**Comment by vilindh** [comment]
Any news on this?
I really don't like that I can edit others' comments, and vice versa - audit trail or not.
Feels strange that this hasn't been adressed by someone from Github 😕

**Comment by gigatesseract** [comment]
+1, feels very weird, I like to understand why it was designed that way

**Comment by mball-agathos** [comment]
I believe that editing other people's comments is only available to admins, but even so, I'm curious if there's a setting somewhere to restrict this behavior.  Claude Code used this permission on my behalf to edit a bunch of PR comments instead of replying to them!

**Comment by Invertex** [comment]
Just had this happen by accident, I assumed I must have clicked quote reply, so I deleted the quoted information and wrote my comment. Turns out I was in edit mode. That was way too easy to happen by accident. There should be a warning that you're editing someone else's comment...

Worst of all, I can't even undo my revision, yet it's showing me a button to remove the original comment, saying "remove revision"... BUT THAT'S NOT A REVISION, it's literally the initial "comment" data, not a revision of it. That makes absolutely no sense.

**Comment by prafulAnand724** [comment]
is this behaviour intentional ??
